test/documentation/api-stability.js

```diff
@@ -13,12 +13,28 @@ $(document).ready(function()
         $(this).find('div.memproto').html(new_html);
     }
 
-    var add_labels = function(api_type, entry_type)
+    var add_labels = function(api_type, api_color, entry_obj, entry_type)
     {
         if (entry_type == "member")
+        {
             $(this).find('span.mlabels').prepend('<span class="mlabel ' + api_type + '">' + api_type + '</span>');
+
+            // get header above the item, append the api_type tag to the header and change the bg color
+            prev = entry_obj;
+            do prev = prev.previousSibling; while(prev && prev.nodeType !== 1);
+
+            prev.innerContent = prev.innerContent + '  ' + '[' + api_type + '] ';
```

Comment:
if you can you should avoid something like this, as this needs to recreate all node. You should be able to append a new "<span>" element.

something like this:
```
$(prev).append();
```

Reply:
but I need to append WITHIN the title (<h2>). `append` would append something after the closing `</h2>` element. So how to I get the inner stuff? I found only this. (And `innerHTML` but that was discouraged and `innerContent was recommended instead`)